assets/konnectivity/konnectivity-server-deployment.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: konnectivity-server
+  labels:
+    app: konnectivity-server
+    clusterID: {{ .ClusterID }}
+spec:
+  selector:
+    matchLabels:
+      app: konnectivity-server
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 3
```

Comment:
It doesn't make sense to have a maxSurge of 3 if there can only be 1 replica.  I think the update strategy could simply be the default (see below) if there can only be 1 replica.

```
  strategy:
    type: RollingUpdate
```

Reply:
changed